openchs-server-api/src/main/java/org/openchs/web/request/rules/constant/WorkFlowTypeEnum.java

```diff
@@ -0,0 +1,8 @@
+package org.openchs.web.request.rules.constant;
+
+public enum WorkFlowTypeEnum {
```

Comment:
enum values should be named ProgramEncounter, Encounter, ProgramEnrolment, Individual for readability. It's fine to expect the same values from API as well.

Reply:
Enums are constant and should be defined in Capitalise.
If you want I will change it to Camel-Case.

Reply:
Agree but let's add underscores to separate the words to make it more readable. PROGRAM_ENCOUNTER, ENCOUNTER, PROGRAM_ENROLMENT, INDIVIDUAL.